src/sdk/python/src/iotea/core/rules.py

```diff
@@ -82,10 +82,17 @@ def stringify_value_type(value_type):
 
 
 class SchemaConstraint(Constraint):
-    def __init__(self, feature, value, type_selector, value_type, path=Constraint.PATH_IDENTITY, instance_id_filter=Constraint.ALL_INSTANCE_IDS_FILTER, limit_feature_selection=True, sid=uuid.uuid1().hex):
+    def __init__(self, feature, value, type_selector, value_type, path=Constraint.PATH_IDENTITY, instance_id_filter=Constraint.ALL_INSTANCE_IDS_FILTER, limit_feature_selection=True, sid=None):
```

Comment:
this files contains very long lines.. can we split it to multiple lines to improve readability?

Reply:
Will be fixed